### PR TITLE
aii-pxelinux: Include  configure hook

### PR DESCRIPTION
**Pull request by alvarosimon**
This PR includes a new `Configure` hook required by aii-opennebula. https://github.com/quattor/aii/pull/112

### CODE REVIEW

**Comment by jrha** [comment]
LGTM.

File: aii-pxelinux/src/test/perl/pxelinux_ks_hooks.t [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+my $fp = "target/test/pxelinux";
+my $mockpxe = Test::MockModule->new('NCM::Component::pxelinux');
+$mockpxe->mock('filepath', $fp);
+
+my $ks = NCM::Component::pxelinux->new();
+my $cfg = get_config_for_profile('pxelinux_ks_hooks');
+
+NCM::Component::pxelinux::pxeprint($cfg);
+$ksuserhooks = NCM::Component::pxelinux::ksuserhooks($cfg, CONFIGURE_HOOK_PATH);
+is($ksuserhooks, undef, 'Configure hook');
```
Comment (stdweird) [inline aii-pxelinux/src/test/perl/pxelinux_ks_hooks.t]: what is this supposed to test?

Comment (alvarosimon) [inline aii-pxelinux/src/test/perl/pxelinux_ks_hooks.t]: @stdweird I was trying to test each hook. Probably this is not the best way to do it..I don't know, I didn't find any previous unitest for these hooks

Comment (stdweird) [inline aii-pxelinux/src/test/perl/pxelinux_ks_hooks.t]: open an issue to write improved unittests for the AII hooks, not ot delay this PR
@jrha ok with this?

**Comment by alvarosimon** [comment]
opened new issue to test aii hooks properly: https://github.com/quattor/aii/issues/116

File: aii-pxelinux/src/test/perl/pxelinux_ks_hooks.t (hunk 1 above)
Comment (jrha) [inline aii-pxelinux/src/test/perl/pxelinux_ks_hooks.t]: Yes, sounds sensible.

**Comment by stdweird** [comment]
@alvarosimon can you make a PR that updates the documentation?

**Comment by alvarosimon** [comment]
@stdweird sure, I will

**Comment by alvarosimon** [comment]
@stdweird the new doc PR is here #123

**Pull request merged**